src/validators/unixdatetime.ts

```diff
@@ -133,10 +133,12 @@ export abstract class UnixDateTimeValidator<O = never> extends ValidatorBase<str
         return typeStr
       }
       case 'rust': {
-        throw new Error('Rust not supported yet')
+        const typeStr = 'std::time::Instant'
```

Comment:
You can't both use std::time::Instant for string dates and number dates, what does Rust serilize it to by default?